chef/data_bags/crowbar/migrate/network/101_add_ironic_network.rb

```diff
@@ -0,0 +1,20 @@
+def upgrade(ta, td, a, d)
+  unless a["networks"].key? "ironic"
+    a["networks"]["ironic"] = ta["networks"]["ironic"]
+  end
+
+  # copy intf1 as intf3
+  a["conduit_map"].each do |cm|
+    unless cm["conduit_list"].key? "intf3"
+      cm["conduit_list"]["intf3"] = cm["conduit_list"]["intf1"]
```

Comment:
Since ```admin``` and ```ironic``` are on a native VLAN (```use_vlan=false```) does it really work to put them on the same conduit? I suppose the ```ironic``` network is used for PXE booting bare-metal instances?